targets/trusty/debian/control

```diff
@@ -0,0 +1,15 @@
+Source: openhim-mediator-mhero
+Maintainer: Martin Brocker <martin@jembi.org>
+Section: web
+Priority: optional
+Standards-Version: 3.9.1
+Build-Depends: debhelper (>= 7)
+Homepage: https://github.com/jembi/openhim-mediator-mhero
+
+
+Package: openhim-mediator-mhero
+Architecture: amd64
+Depends: openhim-core, nvm
```

Comment:
I think this actually need to be `openhim-core-js`. Also I'm not sure if there is a `nvm` package in the ubuntu repositories. Either way, this should be installed by the `openhim-core-js` package I think.